### DISCUSSION TITLE
Journey to Entorus

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#527] [Games]
# Journey to Entorus

Build retro games using WebAssembly for a fantasy console

https://wasm4.org/play/journey-to-entorus

**Comment by maxcurzi** [comment]
Loved the music! I wasn't expecting some of those sounds in WASM4 and the sync with the boss movements are spot-on.

It would be nice to have an option for controlling the heading/shooting with the mouse as the boss fights are hard and every extra hit count. 

Well done!
